spec/index.bs

```diff
@@ -13732,10 +13732,12 @@ In WebGPU, it does so by [$Replace the drawing buffer|replacing the drawing buff
 This enum specifies how color values are displayed to the screen.
 
 <dl dfn-type=enum-value dfn-for=GPUCanvasToneMappingMode>
-    : <dfn>"clamp-to-standard"</dfn>
+    : <dfn>"standard"</dfn>
     ::
-        Clamp color to the standard dynamic range of the screen. This is accomplished by
-        clamping color values in the color space of the screen to the `[0, 1]` interval.
+        Color values within the standard dynamic range of the screen are unchanged and
+        all other color values are projected to the standard dynamic range of the screen.
+        This projection is often accomplished by clamping color values in the color space
```

Comment:
This will make it non-normative:
```suggestion

        Note:
        This projection is often accomplished by clamping color values in the color space
```